lib/config/config-file.js

```diff
@@ -286,9 +286,24 @@ function writeYAMLConfigFile(config, filePath) {
 function writeJSConfigFile(config, filePath) {
     debug(`Writing JS config file: ${filePath}`);
 
-    const content = `module.exports = ${stringify(config, { cmp: sortByKey, space: 4 })};`;
+    let contentToWrite;
+    const stringifiedContent = `module.exports = ${stringify(config, { cmp: sortByKey, space: 4 })};`;
 
-    fs.writeFileSync(filePath, content, "utf8");
+    try {
```

Comment:
I'm confused about the motivation for the try-catch block here. Is there a particular error we're expecting to catch? If so, could anything be moved out of the `try` block to reduce the risk of hiding unexpected errors?

Reply:
I just wanted to make sure a config file still got written in the case `executeOnText` threw an error. I figured writing a config file that didn't conform to the linting rules was better than displaying an error to the user. As I mentioned, this is my first contribution to this repo, so my knowledge of the possible errors is limited. If you think the try-catch is unnecessary, we can remove it.

Reply:
I'm fine with still writing a config file as a fallback if there's an unexpected error, but I think we should still throw the error and/or print a stack trace if that happens. (We could also print a message along the lines of "something went wrong but a config was generated anyway" if that would alleviate confusion.)

Reply:
I also agree that displaying an error message here would be useful for both us and the user. I really like @not-an-aardvark's idea of telling the user that something went wrong but making it clear that we still generated a config file. This route would allow the user to notice/report the error (helpful for us) while still allowing them to get up and running as fast as possible (helpful for them).